### PR TITLE
[Test] testcase with tf-lite ver1.x

### PR DESCRIPTION
**Pull request by jaeyun-jung**
We will remove tf-lite ver1.x on tizen release, fix related testcase.

### CODE REVIEW

File: tests/capi/unittest_capi_inference_single.cc [hunk 1]
```diff
@@ -4157,9 +4157,9 @@ TEST (nnstreamer_capi_ml_option, tensorflow_lite)
   g_free (test_model);
 }
 
-#if defined(ENABLE_TENSORFLOW_LITE) && defined(ENABLE_TENSORFLOW2_LITE)
+#if defined(ENABLE_TENSORFLOW_LITE) || defined(ENABLE_TENSORFLOW2_LITE)
 /**
- * @brief Test ml_option with tensorflow1-lite (manually set by ml_option_se, NULLt)
+ * @brief Test ml_option with tensorflow-lite (manually set by ml_option_set, framework_name=tensorflow-lite)
  */
 TEST (nnstreamer_capi_ml_option, tensorflow1_lite)
```
Comment (anyj0527) [inline tests/capi/unittest_capi_inference_single.cc:4164]: may change the TC name too

Comment (jaeyun-jung) [reply tests/capi/unittest_capi_inference_single.cc]: updated

**Review by anyj0527 — APPROVED** [review]
LGTM✔

**Review by gichan-jang — APPROVED** [review]
LGTM

**Pull request merged**